### PR TITLE
Fix 604 d

### PR DESCRIPTION
**Pull request by huard**
PR interne pour de nouveaux indicateurs. Travis va réviser fix-604-b d'un seul coup.

### CODE REVIEW

File: xclim/indices/_hydrology.py [hunk 1]
```diff
@@ -90,3 +92,76 @@ def rb_flashiness_index(q: xarray.DataArray, freq: str = "YS"):  # noqa: D401
...
+    """
+
+    # Compute change in SWE. Set melt as a positive change.
+    dswe = swe.diff(dim="time") * -1
+
+    # Sum over window
+    agg = dswe.rolling(time=window).sum()
+
+    # Max over period
+    out = agg.max(dim="time")
```
Comment (aulemahal) [inline xclim/indices/_hydrology.py:125]: Wouldn't that be:
```suggestion
    out = agg.resample(time=freq).max(dim="time")
```

File: xclim/indices/_hydrology.py [hunk 2]
```diff
@@ -90,3 +92,76 @@ def rb_flashiness_index(q: xarray.DataArray, freq: str = "YS"):  # noqa: D401
...
+    dswe = swe.diff(dim="time") * -1
+
+    # Add precipitation total
+    total = rate2amount(pr) + dswe
+
+    # Sum over window
+    agg = total.rolling(time=window).sum()
+
+    # Max over period
+    out = agg.max(dim="time")
```
Comment (aulemahal) [inline xclim/indices/_hydrology.py:165]: Same
```suggestion
    out = agg.resample(time=freq).max(dim="time")
```

File: xclim/locales/fr.json [hunk 3]
```diff
@@ -727,11 +727,35 @@
...
+  },
+  "SNOW_MELT_WE_MAX": {
+    "long_name": "Fonte de neige maximale",
+    "description": "Fonte maximale {freq:m} de neige sur {window} jours.",
+    "title": "Fonte de neige maximale",
+    "abstract": "Équivalent en eau de la fonte de neige maximale."
+  },
+  "MELT_AND_PRECIP_MAX": {
+    "long_name": "Maximum des apports en eau incluant la fonte de neige et les précipitations",
+    "description": "Precipitation et fonte de neige maximale {freq:m} sur {window} jours.",
```
Comment (aulemahal) [inline xclim/locales/fr.json:744]:
```suggestion
    "description": "Precipitation et fonte de neige maximales {freq:fpl} sur {window} jours.",
```

File: xclim/locales/fr.json [hunk 4]
```diff
@@ -727,11 +727,35 @@
...
   },
+  "SND_MAX_DOY": {
+    "long_name": "Date de la profondeur de neige maximale",
+    "description": "Date {freq:m} à laquelle la profondeur de neige atteint sa valeur maximale.",
+    "title": "Date où la profondeur de neige est maximale",
+    "abstract": "Jour de l'année où la profondeur de neige atteint sa valeur maximale."
+  },
+  "SNOW_MELT_WE_MAX": {
+    "long_name": "Fonte de neige maximale",
+    "description": "Fonte maximale {freq:m} de neige sur {window} jours.",
```
Comment (aulemahal) [inline xclim/locales/fr.json:738]:
```suggestion
    "description": "Fonte maximale {freq:f} de neige sur {window} jours.",
```

File: xclim/locales/fr.json [hunk 5]
```diff
@@ -727,11 +727,35 @@
     "title": "Date du fin du couvert de neige",
     "abstract": "Première date à partir de laquelle la profondeur de neige est inférieure à un seuil pendant un nombre de jours consécutifs suite à l'établissement du couvert de neige."
   },
+  "SND_MAX_DOY": {
+    "long_name": "Date de la profondeur de neige maximale",
+    "description": "Date {freq:m} à laquelle la profondeur de neige atteint sa valeur maximale.",
```
Comment (aulemahal) [inline xclim/locales/fr.json:732]:
```suggestion
    "description": "Date {freq:f} à laquelle la profondeur de neige atteint sa valeur maximale.",
```

File: xclim/locales/fr.json [hunk 6]
```diff
@@ -727,11 +727,35 @@
...
+  "SNOW_MELT_WE_MAX": {
+    "long_name": "Fonte de neige maximale",
+    "description": "Fonte maximale {freq:m} de neige sur {window} jours.",
+    "title": "Fonte de neige maximale",
+    "abstract": "Équivalent en eau de la fonte de neige maximale."
+  },
+  "MELT_AND_PRECIP_MAX": {
+    "long_name": "Maximum des apports en eau incluant la fonte de neige et les précipitations",
+    "description": "Precipitation et fonte de neige maximale {freq:m} sur {window} jours.",
+    "title": "Précipitations et fonte de neige maximale",
```
Comment (aulemahal) [inline xclim/locales/fr.json:745]:
```suggestion
    "title": "Précipitations et fonte de neige maximales",
```

File: xclim/locales/fr.json [hunk 7]
```diff
@@ -727,11 +727,35 @@
...
+    "abstract": "Maximum des apports en eau provenant des précipitations et de la fonte de neige."
+  },
+  "HIGH_PRECIP_LOW_TEMP": {
+    "long_name": "Jours avec des précipitations au dessus d'un seuil et des températures sous un seuil",
+    "description": "Nombre {freq:m} de jours avec précipitation plus grande ou égales à {pr_thresh} et température inférieure à {tas_thresh}.",
```
Comment (aulemahal) [inline xclim/locales/fr.json:750]:
```suggestion
    "description": "Nombre {freq:m} de jours avec précipitation plus grande ou égale à {pr_thresh} et température inférieure à {tas_thresh}.",
```

File: xclim/locales/fr.json [hunk 8]
```diff
@@ -727,11 +727,35 @@
...
+    "title": "Jours avec précipitations et températures froides"
```
Comment (aulemahal) [inline xclim/locales/fr.json:751]:
```suggestion
    "title": "Jours avec précipitations et températures froides",
```

**Review by aulemahal — APPROVED** [review]
I believe a few  `resample`s were forgotten?
Also suggested a few grammar fixes.

**Comment by huard** [comment]
Merci !

**Pull request merged**